app/services/csv_exporter.rb

```diff
@@ -1,9 +1,10 @@
 require 'csv'
 
 module CsvExporter
-  def self.export(resources, columns)
+  def self.export(resources, columns, header = default_header(columns))
+    raise ArgumentError, "Columns and header row aren't the same length" unless columns.length == header.length
```

Comment:
This error shouldn't ever occur since the parameters are always hard coded at the moment in the callers. It's more of a defensive programming check to make sure the developers don't miscount the number of columns and create invalid CSVs.